docs/src/main/paradox/external-connectors.md

```diff
@@ -1,8 +1,10 @@
 # External Connectors
 
-Connectors that are maintained by other projects.
+[![alpakka]][alpakka-scaladex] Feel free to tag your project with *alpakka* keyword in Scaladex for easier discoverability.
```

Comment:
Hm. I am not sure. The management page for a project in scaladex is quite self explanatory.